asset/CMakeLists.txt

```diff
@@ -5,6 +5,15 @@ if(APPLE)
 	SET(ASSETS ${ASSETS}
 		${CMAKE_CURRENT_SOURCE_DIR}/brd.icns
 	)
+elseif(UNIX)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ofbv-app.svg
+		DESTINATION share/icons/hicolor/scalable/apps RENAME openboardview.svg)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.appdata.xml
+		DESTINATION share/appdata)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.desktop
+		DESTINATION share/applications)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.xml
+		DESTINATION share/mime/packages)
```

Comment:
Well if you are installing under `/usr/local`, you would want to install the files under `/usr/local/share`, as well?